Dockerfile.j2

```diff
@@ -25,30 +34,31 @@ RUN apt update \
 RUN add-apt-repository -y ppa:fcwu-tw/apps \
     && apt update \
     && apt install -y --no-install-recommends --allow-unauthenticated \
-        lxde xvfb x11vnc=0.9.16-1 \
-        firefox chromium-browser \
-        ttf-ubuntu-font-family ttf-wqy-zenhei \
+        xvfb x11vnc=0.9.16-1 \
+        {%for package in addon_packages%}{{package}} {%endfor%} \
     && add-apt-repository -r ppa:fcwu-tw/apps \
     && apt autoclean -y \
     && apt autoremove -y \
     && rm -rf /var/lib/apt/lists/*
+{%if desktop == "lxde" %}
 RUN apt update \
     && apt install -y --no-install-recommends --allow-unauthenticated \
-        gtk2-engines-murrine gnome-themes-standard gtk2-engines-pixbuf gtk2-engines-murrine arc-theme \
+        lxde gtk2-engines-murrine gnome-themes-standard gtk2-engines-pixbuf gtk2-engines-murrine arc-theme \
     && apt autoclean -y \
     && apt autoremove -y \
     && rm -rf /var/lib/apt/lists/*
+{%endif%} 
 # Additional packages require ~600MB
 # libreoffice  pinta language-pack-zh-hant language-pack-gnome-zh-hant firefox-locale-zh-hant libreoffice-l10n-zh-tw
 
-# tini for subreap                                   
+# tini for subreap
 ARG TINI_VERSION=v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-armhf /bin/tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /bin/tini
```

Comment:
Should it be tini-${ARCH}?

Reply:
Yes, you are correct, I did miss that one. I do not  have an arm machine at hand so I only did a test with amd64.

This is just an idea, or a suggestion. If you like it I would add more flavors.